test/sql/order_by_sql_test.cpp

```diff
@@ -98,5 +109,348 @@ TEST_F(OrderBySQLTests, PerformanceTest) {
+  txn_manager.CommitTransaction(txn);
+}
+
+
+TEST_F(OrderBySQLTests, OrderByStar) {
+  catalog::Catalog::GetInstance()->CreateDatabase(DEFAULT_DB_NAME, nullptr);
+
+  CreateAndLoadTable();
+
+  std::vector<ResultType> result;
+  std::vector<FieldInfoType> tuple_descriptor;
+  std::string error_message;
+  int rows_changed;
+
+  SQLTestsUtil::ExecuteSQLQuery("SELECT * FROM test ORDER BY d;", result, tuple_descriptor, rows_changed, error_message);
+
+  // Check the return value
+  // Should be: [1, 22, 333, 'abcd']; [3,...], [2,...];
+  EXPECT_EQ(0, rows_changed);
+  EXPECT_EQ("1", SQLTestsUtil::GetResultValueAsString(result, 0));
+  EXPECT_EQ("22", SQLTestsUtil::GetResultValueAsString(result, 1));
+  EXPECT_EQ("333", SQLTestsUtil::GetResultValueAsString(result, 2));
+  EXPECT_EQ("abcd", SQLTestsUtil::GetResultValueAsString(result, 3));
+  EXPECT_EQ("3", SQLTestsUtil::GetResultValueAsString(result, 4));
+  EXPECT_EQ("2", SQLTestsUtil::GetResultValueAsString(result, 8));
+
+  // free the database just created
+  auto &txn_manager = concurrency::TransactionManagerFactory::GetInstance();
+  auto txn = txn_manager.BeginTransaction();
+  catalog::Catalog::GetInstance()->DropDatabaseWithName(DEFAULT_DB_NAME, txn);
+  txn_manager.CommitTransaction(txn);
+}
+
+TEST_F(OrderBySQLTests, OrderByStarDesc) {
+  catalog::Catalog::GetInstance()->CreateDatabase(DEFAULT_DB_NAME, nullptr);
+
+  CreateAndLoadTable();
+
+  std::vector<ResultType> result;
+  std::vector<FieldInfoType> tuple_descriptor;
+  std::string error_message;
+  int rows_changed;
+
+  SQLTestsUtil::ExecuteSQLQuery("SELECT * FROM test ORDER BY d DESC;", result, tuple_descriptor, rows_changed, error_message);
+
+  // Check the return value
+  // Should be: [2, 33, 111, 'bcda']; [3,...], [1,...];
+  EXPECT_EQ(0, rows_changed);
+  EXPECT_EQ("2", SQLTestsUtil::GetResultValueAsString(result, 0));
+  EXPECT_EQ("33", SQLTestsUtil::GetResultValueAsString(result, 1));
+  EXPECT_EQ("111", SQLTestsUtil::GetResultValueAsString(result, 2));
+  EXPECT_EQ("bcda", SQLTestsUtil::GetResultValueAsString(result, 3));
+  EXPECT_EQ("3", SQLTestsUtil::GetResultValueAsString(result, 4));
+  EXPECT_EQ("1", SQLTestsUtil::GetResultValueAsString(result, 8));
+
+  // free the database just created
+  auto &txn_manager = concurrency::TransactionManagerFactory::GetInstance();
+  auto txn = txn_manager.BeginTransaction();
+  catalog::Catalog::GetInstance()->DropDatabaseWithName(DEFAULT_DB_NAME, txn);
+  txn_manager.CommitTransaction(txn);
+}
+
+TEST_F(OrderBySQLTests, OrderByStarWithLimit) {
+  catalog::Catalog::GetInstance()->CreateDatabase(DEFAULT_DB_NAME, nullptr);
+
+  CreateAndLoadTable();
+
+  std::vector<ResultType> result;
+  std::vector<FieldInfoType> tuple_descriptor;
+  std::string error_message;
+  int rows_changed;
+
+  SQLTestsUtil::ExecuteSQLQuery("SELECT * FROM test ORDER BY d LIMIT 2;", result, tuple_descriptor, rows_changed, error_message);
+
+
```

Comment:
Look at this[https://github.com/cmu-db/peloton/wiki/Formatting]( https://github.com/cmu-db/peloton/wiki/Formatting)